### ISSUE TITLE
Need proper message to convey that task does not have participants

### ISSUE THREAD

**Issue by ankurnarkhede** [Real-Dev-Squad/website-status#87]
Currently, the UI shows an alt text value, it should show a message or anything different converting that the task does not have any participants.

<img width="757" alt="Screenshot 2021-03-18 at 11 45 15 PM" src="https://user-images.githubusercontent.com/22790328/111676382-1c5de000-8844-11eb-8733-7fc488e35196.png">

**Comment by whyDontI** [comment]
Thanks, closing the issue.
![Screenshot from 2021-04-14 19-41-44](https://user-images.githubusercontent.com/26569942/114724725-7e353b00-9d59-11eb-8a1f-8575a9fd993c.png)
